webui.py

```diff
@@ -621,8 +624,31 @@ def save_sample(image, sample_path_i, filename, jpg_sample, prompts, seeds, widt
             #info_dict["init_mask"] = init_mask
             info_dict["denoising_strength"] = denoising_strength
             info_dict["resize_mode"] = resize_mode
-        with open(f"{filename_i}.yaml", "w", encoding="utf8") as f:
-            yaml.dump(info_dict, f, allow_unicode=True, width=10000)
+        if write_info_files:
+            with open(f"{filename_i}.yaml", "w", encoding="utf8") as f:
+                yaml.dump(info_dict, f, allow_unicode=True, width=10000)
+
+        if write_info_to_one_file:
+            ignore_list = ["prompt", "target", "toggles", "ddim_eta", "batch_size"]
+            rename_dict = {"ddim_steps": "steps", "n_iter": "number", "sampler_name": "sampler"} #changes the name of parameters to match with dynamic parameters
+            sample_log_path = os.path.join(sample_path_i, "log.yaml")

```

Comment:
Do not use the ".yaml" file ending for log files, that will confuse users trying to load settings from them using https://github.com/hlky/stable-diffusion-webui/pull/436 .
Use a file name that ends with ".txt" or ".log" instead.